scripts/apps/authoring-react/authoring-react.tsx

```diff
@@ -607,8 +607,8 @@ export class AuthoringReact<T extends IBaseRestApiResponse> extends React.PureCo
             addInternalEventListener(
                 'dangerouslyOverwriteAuthoringData',
                 (event) => {
-                    if (event.detail._id === this.props.itemId) {
-                        const patch = event.detail;
+                    if (event.detail.item._id === this.props.itemId) {
```

Comment:
is applying `setDirty` not needed here?

Reply:
No, that's why I've ignored it

Reply:
ok just wanted to check if you didn't forget